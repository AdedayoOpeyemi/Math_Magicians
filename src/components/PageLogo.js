import React from 'react';
import '../styles/pagelogo.css';

const PageLogo = () => (
  <h1>Math Magicians</h1>
);
export default PageLogo;
