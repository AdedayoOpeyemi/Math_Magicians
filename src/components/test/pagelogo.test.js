import renderer from 'react-test-renderer';
import PageLogo from '../PageLogo';

it('PageLogo renders correctly', () => {
  const tree = renderer
    .create(<PageLogo />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
