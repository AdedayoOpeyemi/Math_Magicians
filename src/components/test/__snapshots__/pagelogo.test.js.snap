// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLogo renders correctly 1`] = `
<h1>
  Math Magicians
</h1>
`;

exports[`renders correctly 1`] = `
<nav
  className="navBar"
>
  <h1>
    Math Magicians
  </h1>
  <ul>
    <li>
      <a
        aria-current="page"
        className="active-link"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
